inference-engine/tests_deprecated/functional/vpu/common/layers/myriad_layers_convolution3x3.hpp

```diff
@@ -102,7 +102,7 @@ void refConvolution3x3(const Blob::Ptr src, InferenceEngine::TBlob<uint8_t>::Ptr
     }
 }
 
-TEST_P(myriadConvolution3x3LayerTests_smoke, Convolution3x3) {
+TEST_P(myriadConvolution3x3LayerTests_smoke, DISABLED_Convolution3x3) {
```

Comment:
Nitpick: it's better to mention ticket in the code while disabling tests

Also, it's a good option to use single place for disabled tests, e.g. `openvino/inference-engine/tests_deprecated/functional/vpu/myriad_tests/vpu_tests_config.cpp`, but since we're not using that, I guess we will refactor all disabled deprected tests at once.